label_studio/core/views.py

```diff
@@ -169,6 +170,21 @@ def samples_paragraphs(request):
     return HttpResponse(json.dumps(result), content_type='application/json')
 
 
+def heidi_tips(request):
```

Comment:
any way we can make this optional? I'm concerned if an outage happens this will block pytest checks

Reply:
Great point, I'm going to look at that because honestly we don't want this in any pytests as there is nothing to test but everything to possibly fail randomly which is not helpful.

Reply:
At the very least we'd want to mock the requests call.

Reply:
Double checked it, we don't run these urls in tests, so this is not being executed in the tests currently. Also, the handling of this is such that there is a max timeout of 5 seconds and the call made for this file will also gracefully degrade for air-gapped environments with no internet access. So in all cases this will never block and either return the response code from the error thrown from fetching the file, or 404 in the case the file cannot be reached or there are network issues.